configure.py

```diff
@@ -1168,62 +1171,62 @@ def configure_library(lib, output):
+  o['variables']['v8_use_snapshot'] = 0 if args.without_snapshot else 1
+  o['variables']['v8_trace_maps'] = 1 if args.trace_maps else 0
+  o['variables']['node_use_v8_platform'] = b(not args.without_v8_platform)
+  o['variables']['node_use_bundled_v8'] = b(not args.without_bundled_v8)
+  o['variables']['force_dynamic_crt'] = 1 if args.shared else 0
+  o['variables']['node_enable_d8'] = b(args.enable_d8)
+  if args.enable_d8:
     o['variables']['test_isolation_mode'] = 'noop'  # Needed by d8.gyp.
-  if options.without_bundled_v8 and options.enable_d8:
+  if args.without_bundled_v8 and args.enable_d8:
     raise Exception('--enable-d8 is incompatible with --without-bundled-v8.')
-  if options.without_bundled_v8 and options.build_v8_with_gn:
+  if args.without_bundled_v8 and args.build_v8_with_gn:
     raise Exception(
         '--build-v8-with-gn is incompatible with --without-bundled-v8.')
-  if options.build_v8_with_gn:
+  if args.build_v8_with_gn:
     v8_path = os.path.join('deps', 'v8')
     print('Fetching dependencies to build V8 with GN')
-    options.build_v8_with_gn = FetchDeps(v8_path)
-  o['variables']['build_v8_with_gn'] = b(options.build_v8_with_gn)
+    args.build_v8_with_gn = FetchDeps(v8_path)
+  o['variables']['build_v8_with_gn'] = b(args.build_v8_with_gn)
 
 
 def configure_openssl(o):
   variables = o['variables']
-  variables['node_use_openssl'] = b(not options.without_ssl)
-  variables['node_shared_openssl'] = b(options.shared_openssl)
-  variables['openssl_is_fips'] = b(options.openssl_is_fips)
+  variables['node_use_openssl'] = b(not args.without_ssl)
+  variables['node_shared_openssl'] = b(args.shared_openssl)
+  variables['openssl_is_fips'] = b(args.openssl_is_fips)
   variables['openssl_fips'] = ''
 
-  if options.openssl_no_asm:
+  if args.openssl_no_asm:
     variables['openssl_no_asm'] = 1
 
-  if options.without_ssl:
+  if args.without_ssl:
     def without_ssl_error(option):
       error('--without-ssl is incompatible with %s' % option)
-    if options.shared_openssl:
+    if args.shared_openssl:
       without_ssl_error('--shared-openssl')
-    if options.openssl_no_asm:
+    if args.openssl_no_asm:
       without_ssl_error('--openssl-no-asm')
-    if options.openssl_fips:
+    if args.openssl_fips:
       without_ssl_error('--openssl-fips')
     return
 
-  if options.use_openssl_ca_store:
+  if args.use_openssl_ca_store:
     o['defines'] += ['NODE_OPENSSL_CERT_STORE']
-  if options.openssl_system_ca_path:
-    variables['openssl_system_ca_path'] = options.openssl_system_ca_path
-  variables['node_without_node_options'] = b(options.without_node_options)
-  if options.without_node_options:
-      o['defines'] += ['NODE_WITHOUT_NODE_OPTIONS']
+  if args.openssl_system_ca_path:
+    variables['openssl_system_ca_path'] = args.openssl_system_ca_path
+  variables['node_without_node_args'] = b(args.without_node_args)
+  if args.without_node_args:
+      o['defines'] += ['NODE_WITHOUT_NODE_args']
```

Comment:
This needs to remain `NODE_WITHOUT_NODE_OPTIONS`.

Reply:
Thanks for finding these errors. I think I fixed them all.